rollbar-spring-boot/src/main/java/com/rollbar/springboot/RollbarExceptionResolver.java

```diff
@@ -0,0 +1,36 @@
+package com.rollbar.springboot;
+
+import com.rollbar.notifier.Rollbar;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.springframework.core.Ordered;
+import org.springframework.web.servlet.HandlerExceptionResolver;
+import org.springframework.web.servlet.ModelAndView;
+
+
+public class RollbarExceptionResolver implements HandlerExceptionResolver, Ordered {
+
+  private Rollbar rollbar;
+
+  public RollbarExceptionResolver(Rollbar rollbar) {
+    this.rollbar = rollbar;
+  }
+
+  @Override
+  public ModelAndView resolveException(HttpServletRequest request,
+                                     HttpServletResponse response,
+                                     Object handler,
+                                     Exception ex) {
+    System.out.println("******************* inside RollbarExceptionResolver *******************");
+    rollbar.error(ex);
+
+    // null = run other HandlerExceptionResolvers to actually handle the exception
+    return null;
+  }
+
+  @Override
+  public int getOrder() {
```

Comment:
We may not need this if we specify `@Order(Ordered.HIGHEST_PRECEDENCE)` for the class itself.

Reply:
I am not sure, i kept it to be safe since this is the same call Sentry also uses for their Error handling

Reply:
I didn't write a unit test for @Order (https://github.com/rollbar/rollbar-java/pull/211/files#diff-5849b384e2e652af00fc3403953c96eeR12)... since it will basically be testing Spring Framework rather than our code. I think the unit-test is still valid. 

@mrunalk @waltjones - what do you guys think does this look good now?